tasks/lib/jscs.js

```diff
@@ -138,7 +151,7 @@ exports.init = function( grunt ) {
      * Get inline options
      * @return {Object}
      */
-    JSCS.prototype.getOptions = function() {
+    JSCS.prototype.getInlineOptions = function() {
```

Comment:
We can and probably should consider this as internal API and therefore not bother ourself about semver in this context.

Anyway this is why i added "Strictly speaking"